de/main.po

```diff
@@ -1,815 +1,881 @@
-msgstr ""
+msgstr "Gehe zum Kanal und schreibe:"
 
 msgid "or"
-msgstr ""
+msgstr "oder"
 
 msgid "Alternatively, change the channel's mode"
-msgstr ""
+msgstr "Alternativ kann auch der Modus des Kanals geändert werden"
 
 msgid "(receives translated messages only)"
-msgstr ""
+msgstr "(empfängt nur übersetzte Nachrichten)"
 
 msgid "(sends translated messages only)"
-msgstr ""
+msgstr "(sendet nur übersetzte Nachrichten)"
 
 msgid "Stop User Automatic Translation"
-msgstr ""
+msgstr "Automatische Nutzer-Übersetzung stoppen"
 
 msgid "Ban User from using Translator"
-msgstr ""
+msgstr "Nutzer von der Verwendung des Translators ausschließen"
 
 msgid "Stop translating other Bots"
-msgstr ""
+msgstr "Übersetzen von anderen Bots stoppen"
 
 msgid "Simply remove the `%s` role from the other bots."
-msgstr ""
+msgstr "Entferne einfach die `%s` Rolle von den anderen Bots."
 
 msgid "Ignore specific messages"
-msgstr ""
+msgstr "Bestimmte Nachrichten ignorieren"
 
 msgid "Ignore specific words"
-msgstr ""
+msgstr "Bestimmte Wörter ignorieren"
 
 msgid "Miscellaneous Commands"
-msgstr ""
+msgstr "Verschiedene Befehle"
 
 msgid "Help"
-msgstr ""
+msgstr "Hilfe"
 
 msgid "Info"
-msgstr ""
+msgstr "Info"
 
 msgid "Links"
-msgstr ""
+msgstr "Links"
 
 msgid "Stats"
-msgstr ""
+msgstr "Statistiken"
 
 msgid "(bot invite link)"
-msgstr ""
+msgstr "(Bot Einladungslink)"
 
 msgid "(official server)"
-msgstr ""
+msgstr "(offizieller Server)"
 
 msgid "Detect Language of Message"
-msgstr ""
+msgstr "Sprache der Nachricht erkennen"
 
 msgid ""
 "You can detect the language of a message by reacting to it with a **globe** "
 "emoji (%s).\n"
 "Alternatively, you can use the `%s` command:"
 msgstr ""
+"Du kannst die Sprache einer Nachricht erkennen, indem du darauf mit einem "
+"**globe** Emoji (%s) reagierst.\n"
+"Alternativ kannst du auch den Befehl `%s` verwenden:"
 
 msgid "Auto Delete Bot Commands"
-msgstr ""
+msgstr "Bot-Befehle automatisch löschen"
 
 msgid ""
 "You can choose to have the bot auto delete commands in the channel. All you "
 "have to do is write %s anywhere inside the channel's topic description."
 msgstr ""
+"Du kannst wählen, ob der Bot die Befehle in diesem Kanal automatisch löschen "
+"soll. Alles, was du tun musst, ist, %s irgendwo innerhalb der "
+"Themenbeschreibung des Kanals hinzuzufügen."
 
 msgid ""
 "This will delete all commands/messages including: help commands, settings, "
 "stats and translate commands for example:"
 msgstr ""
+"Dadurch werden alle Befehle/Nachrichten gelöscht, z.B.: Hilfebefehle, "
+"Einstellungen, Statistiken und Übersetzungsbefehle:"
 
 msgid "and"
-msgstr ""
+msgstr "und"
 
 msgid "Translate other Bots"
-msgstr ""
+msgstr "Andere Bots übersetzen"
 
 msgid ""
 "By default, Translator will ignore all messages by bots, this is a common "
 "practice to avoid unpredictable behaviours and response-loops."
 msgstr ""
+"Standardmäßig ignoriert Translator alle Nachrichten von Bots, dies ist eine "
```

Comment:
We should add a dot after `Standardmäßig ignoriert Translator alle Nachrichten von Bots` or use a `;`.